### PR TITLE
Penv 167 consensus deploy tool

### PR DESCRIPTION
**Pull request by SergeyRadist**
**- What I did**
added deploy tool
nginx ingress changed to traefik
added consensus test template in CI
didn't added tool to "make build"
**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,3 +16,29 @@ To build the project run:
...
+        prometheus.io/scrape: "true"
+```
+
+You can specify the port which your app is using. Any new deployed software with correct annotations will be scraped by prometheus.
+
+After prometheus and ingress controller are up and running, you can check what's going on:
+
+To get a list of metrics names and their series count:
+```
+curl -s 'http://localhost/prometheus/api/v1/status/tsdb' | jq .
```
Comment (ElSacramento) [inline README.md:38]: `localhost:8001` I think

File: ledger-core/v2/scripts/deploy/kube-deploy-tool/config-generator.go [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+	"gopkg.in/yaml.v2"
+
+	"github.com/insolar/assured-ledger/ledger-core/v2/application/bootstrap"
+	pulsewatcher "github.com/insolar/assured-ledger/ledger-core/v2/cmd/pulsewatcher/config"
+)
+
+var transportPort = 13831
+var adminAPIPort = 19001
+var hostTemplate = "virtual-%d.platform-network.insolar:%d"
+var certNameTemplate = "virtual-%d.json"
```
Comment (ElSacramento) [inline ledger-core/v2/scripts/deploy/kube-deploy-tool/config-generator.go:21]: It's a public repo, is it secure to show our hosts?

File: ledger-core/v2/scripts/deploy/kube-deploy-tool/config.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+netparams:
+- nodescount: 3
+  waitinready: 30s
+- nodescount: 4
+- nodescount: 5
+kubeparams:
+  env: local
+  kuberootpath: /../scripts/deploy/kube/
+  manifestsrelpath: manifests/
+  bootstrapconfigrelpath: manifests/configuration/
```
Comment (ElSacramento) [inline ledger-core/v2/scripts/deploy/kube-deploy-tool/config.yaml:10]: This param name is really hard to read.

File: README.md (hunk 1 above)
Comment (yellowmegaman) [reply README.md:38]: Nope, it's about access to prometheus itself, not metrics endpoint of any part of platform. And it's defined as Ingress object here ledger-core/v2/scripts/deploy/kube/prometheus/05-prometheus-Ingress.yaml

File: ledger-core/v2/scripts/deploy/kube-deploy-tool/kube-deploy-tool.go [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+			err := insolarManager.collectLogs(net)
+			if err != nil {
+				fmt.Printf("failed to collect logs: %s\n nodes count: %d", err.Error(), net.NodesCount)
+			}
+		}
+
+		if startAndReadyError != nil {
+			fmt.Print(startAndReadyError.Error())
+			if cfg.KubeParams.LeaveInsolarAliveIfFailed {
+				panic("LeaveInsolarAliveIfFailed is true, shutting down")
```
Comment (ElSacramento) [inline ledger-core/v2/scripts/deploy/kube-deploy-tool/kube-deploy-tool.go:89]: `not shutting down`?

Comment (SergeyRadist) [reply ledger-core/v2/scripts/deploy/kube-deploy-tool/kube-deploy-tool.go]: rephrased

**Pull request merged**